tests/expected_traces.py

```diff
@@ -1,78 +1,76 @@
 LOCAL_TRACE = """
```

Comment:
Don't  concern yourself too much with the diff in this file... It is just a place to keep the expected output so that we can isolate flake8 violations and such.

The real important changes are in `test_trace.py`